apps/server/src/modules/tool/context-external-tool/service/tool-reference.service.ts

```diff
@@ -29,11 +33,16 @@ export class ToolReferenceService {
 		);
 		const externalTool: ExternalTool = await this.externalToolService.findById(schoolExternalTool.toolId);
 
-		const status: ToolConfigurationStatus = this.commonToolService.determineToolConfigurationStatus(
-			externalTool,
-			schoolExternalTool,
-			contextExternalTool
-		);
+		let status: ToolConfigurationStatus;
+		if (Configuration.get('FEATURE_COMPUTE_TOOL_STATUS_WITHOUT_VERSIONS_ENABLED')) {
```

Comment:
Please add the feature flag to tool-config.module, inject the toolFeatures and use it